content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst

```diff
@@ -14,9 +14,18 @@ In order to use the Odoo DHL API, you will need:
 Getting SiteID and password for countries other than United States (UK and Rest of the world)
 =============================================================================================
 
-You should contact DHL account manager and request integration for XML Express API. The presales should provide you live credentials.
+You should contact DHL account manager and request integration for XML Express API. The presales
+should provide you live credentials.
 
 Getting SiteID and Password for  United States
 ==============================================
 
-You need to write to xmlrequests@dhl.com along with your full Account details like account number, region, address, etc. to get API Access.
+You need to write to xmlrequests@dhl.com along with your full Account details like account number,
+region, address, etc. to get API Access.
+
+``P - DHL Express Worldwide``
+
+``"ship": {"ShipmentDetails": {"Contents": "Guns and roses"}}``
+
+``"ship": {"ShipmentDetails": {"Contents": "Guns and roses"}, "SpecialService":
+{"SpecialServiceType": "WY"}}``
```

Comment:
Think this top portion might've been added to this PR by accident. May want to consider removing to avoid any issues. Thanks!